lib/system/schubfach.nim

```diff
@@ -95,6 +95,36 @@ proc floorLog2Pow10(e: int32): int32 {.inline.} =
   sf_Assert(e <= 1233)
   return floorDivPow2(e * 1741647, 19)
 
+const
```

Comment:
Why was this required? Workaround a pre-existing IC bug?

Reply:
yes, this is explained here https://github.com/nim-lang/Nim/pull/17978#issuecomment-868892830